README.md

```diff
@@ -88,7 +88,7 @@ To build stylesheets and javascript bundles, run webpack:
 You will need to rebuild static files after you modify JavaScript or CSS. If you want to rebuild
 these source files as you change them then you can run webpack in watch mode:
 
-    ./develop.sh npm run build:dev -- --watch
+    ./develop.sh npm run build:watch
```

Comment:
We have automatic watching and building of javascript assets in LB btw, if you want to go that way, it's definitely very convenient.

Reply:
we talked about this in IRC. In the future if we need constant asset compilation we'll set up a container to build assets like in LB